lib/rentlinx/modules/linkable.rb

```diff
@@ -0,0 +1,38 @@
+module Rentlinx
+  module Linkable
+    def post_with_links
+      post
+      post_links
+    end
+
+    def post_links
+      Rentlinx.client.post_links(@links)
+    end
+
+    def links
+      return @links if @links
```

Comment:
You could remove this line by using <code>@links ||= if defined? unitID …</code>